dotnet-core/plan.sh

```diff
@@ -25,34 +25,33 @@ pkg_deps=(
 pkg_build_deps=(
   core/patchelf
 )
-pkg_bin_dirs=(bin)
+pkg_bin_dirs=(.)
```

Comment:
I see why this has been set to `(.)`, but we should move the binaries that get created into a `bin` directory and set this back to `(bin)`